zip-0316.rst

```diff
@@ -416,6 +437,21 @@ Requirements for both Unified Addresses and Unified Viewing Keys
   that cannot be interpreted as specified above.
 
 
+Adding new types
+----------------
+
+It is intended that new Receiver Types and Viewing Key Types SHOULD be
+introduced either by a modification to this ZIP or by a new ZIP, in accordance
+with the ZIP Process [#zip-0000]_.
+
+Experimental types MAY be added using the reserved Typecodes 0xFFFA to 0xFFFF
+inclusive. This provides for six simultaneous experiments, which can be
+referred to as experiments A to F. This should be sufficient because
+experiments are expected to be reasonably short-term, and should otherwise
+be either standardized in a ZIP (and allocated a Typecode outside this
+reserved range) or discontinued.
+
+
```

Comment:
That was @str4d's idea. Yes I like it too!

Reply:
On reflection I think the wording is clear as-is. There's nothing to stop experimental type users from coordinating to avoid collisions, so it wouldn't be correct to say they should *expect* them.